bin/rule.ml

```diff
@@ -142,7 +144,8 @@ let run () md_file section direction prelude prelude_str root =
     let options =
       List.map (Fmt.to_to_string pp_prelude) prelude @
       List.map (Fmt.to_to_string pp_prelude_str) prelude_str @
-      [Fmt.to_to_string pp_direction direction]
+      [Fmt.to_to_string pp_direction direction] @
+      (match section with Some s -> [Fmt.(to_to_string pp_section) s] | None -> [])
```

Comment:
why is this needed?

Reply:
That's unrelated.
The `-s` option is used by `rule` but not passed to `test`, that will not work if ignored sections need extra deps.
Other options are correctly passed (`prelude`, `direction`, `root`...).

Should I move this commit to an other PR?

Reply:
yes please, let's keep that PR focused on cleaning up the code without any bug fixes.

Reply:
I moved that commit to https://github.com/realworldocaml/mdx/pull/176